next.config.js

```diff
@@ -40,6 +40,12 @@ const nextConfig = {
   // the modules that newrelic supports should not be mangled by webpack. Thus,
   // we need to "externalize" all of the modules that newrelic supports.
 
+  logging: {
+    fetches: {
+      fullUrl: true,
+    },
+  },
```

Comment:
this utility only runs in dev mode, but is handy to see when data is *actually* being fetched